HACKING.md

```diff
@@ -94,4 +96,22 @@ To rebuild the containers after a change, add the `--build` flag to the `docker-
 
 ```
 docker-compose up --build
-```
\ No newline at end of file
+```
+
+## Shortening the loop
+
+For some components, it is possible to install distribution packages first and then only to replace binaries which may or may not work for smaller changes.
+
+```
+systemctl stop osbuild-composer.service osbuild-composer.socket osbuild-local-worker.socket
+go build ./cmd/osbuild-composer && cp osbuild-composer /usr/libexec/osbuild-composer/
```

Comment:
TEENY TINY NITPICK: we have a `build` target in the `Makefile` that builds all the binaries under `bin/`.  It's annoying to use in this case because it builds a lot more binaries than we need here, but maybe it'd be nice for consistency to do `go build -o bin/ ...`.  That would also remove the need for adding the binaries to `.gitignore`.

And then also using `sudo install -m 755 ...` instead of `cp` to make sure perms are correct (the user's umask might have group and other modes blank, for example).

Not really necessary though.  We can change it later.

Reply:
Done